Source/Shaders/SkyAtmosphereFS.glsl

```diff
@@ -52,6 +75,17 @@ void main (void)
     
     vec3 rgb = rayleighPhase * v_rayleighColor + miePhase * v_mieColor;
     rgb = vec3(1.0) - exp(-exposure * rgb);
+    // compute luminance before color correction to avoid strangely gray night skies
     float l = czm_luminance(rgb);
+
+    // convert rgb color to hsv
+    vec3 hsv = rgb2hsb(rgb);
+    // perform hsv shift
+    hsv.x += u_hsvShift.x; // hue
+    hsv.y = clamp(hsv.y + u_hsvShift.y, 0.0, 1.0); // saturation
+    hsv.z = hsv.z > epsilon ? hsv.z + u_hsvShift.z : 0.0; // brightness
+    // convert shifted hsv back to rgb
+    rgb = hsb2rgb(hsv);
```

Comment:
Since every combination of options doubles the number of compiled shaders, should I only use ifdefs for the color correction for now? The extra calculation for dynamic atmosphere color is just a vec4 subtraction.